.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: groovy
```

Comment:
Since our project is JavaScript primarily we don't want to have ```language: groovy ``` just because we want to run checkstyle. 

Instead would have ```language: node_js``` defined like [here](https://github.com/babel/babel/blob/master/.travis.yml#L2). Since it is impossible to have more that one language defined for Travis build we would need to either run node command from gradle (which would be an overkill) or run gradle from node.js